### DISCUSSION TITLE
[Feature Request] Safari Extension for News Feed Eradicator

### DISCUSSION THREAD

**Discussion by bigplayer-ai** [jordwest/news-feed-eradicator#407] [Ideas]
[Feature Request] Please consider developing a Safari Extension for the News Feed Eradicator.

While it would be ideal for this extension to be exclusive to macOS, it would be even better if its compatibility could be extended to other Apple devices.

**Comment by Rasinari** [comment]
I agree with you. Please create a version for Safari Mac. It is particularly necessary there, as there are no working alternatives, at least those that can be tested beforehand.
You don't need an Apple account for this. Thank you.
Make dev is not working with Terminal on a Mac.
